nextroute/schema/input.go

```diff
@@ -31,54 +31,54 @@ type VehicleDefaults struct {
 	StartLevel              any        `json:"start_level,omitempty"`
 	StartLocation           *Location  `json:"start_location,omitempty"`
 	EndLocation             *Location  `json:"end_location,omitempty"`
-	Speed                   *float64   `json:"speed,omitempty"`
+	Speed                   *float64   `json:"speed,omitempty" minimumExclusive:"0"`
 	StartTime               *time.Time `json:"start_time,omitempty"`
 	EndTime                 *time.Time `json:"end_time,omitempty"`
-	MinStops                *int       `json:"min_stops,omitempty"`
-	MinStopsPenalty         *float64   `json:"min_stops_penalty,omitempty"`
-	MaxStops                *int       `json:"max_stops,omitempty"`
-	MaxDistance             *int       `json:"max_distance,omitempty"`
-	MaxDuration             *int       `json:"max_duration,omitempty"`
-	MaxWait                 *int       `json:"max_wait,omitempty"`
-	CompatibilityAttributes *[]string  `json:"compatibility_attributes,omitempty"`
-	ActivationPenalty       *int       `json:"activation_penalty,omitempty"`
-	AlternateStops          *[]string  `json:"alternate_stops,omitempty"`
+	MinStops                *int       `json:"min_stops,omitempty" minimum:"0"`
+	MinStopsPenalty         *float64   `json:"min_stops_penalty,omitempty" minimum:"0"`
+	MaxStops                *int       `json:"max_stops,omitempty" minimumExclusive:"0"`
+	MaxDistance             *int       `json:"max_distance,omitempty" minimumExclusive:"0"`
```

Comment:
it can't be 0: https://github.com/nextmv-io/platform/blob/2d7a22f5c531adab93c11e146e2d52c3bc3adbf1/core/plugins/nextroute/factory/constraint_distance_limit.go#L37

Reply:
We should not throw an error when passing a perfectly feasible limit....